pkg/app/http.go

```diff
@@ -346,13 +349,15 @@ func (h *Handler) makeAppJS() []byte {
 	if err := template.
 		Must(template.New("app.js").Parse(appJS)).
 		Execute(&b, struct {
-			Env      string
-			Wasm     string
-			WorkerJS string
+			Env            string
+			Wasm           string
+			WorkerJS       string
+			UpdateInterval string
 		}{
-			Env:      btos(env),
-			Wasm:     h.Resources.AppWASM(),
-			WorkerJS: h.resolvePackagePath("/app-worker.js"),
+			Env:            btos(env),
+			Wasm:           h.Resources.AppWASM(),
+			WorkerJS:       h.resolvePackagePath("/app-worker.js"),
+			UpdateInterval: strconv.Itoa(h.UpdateInterval),
```

Comment:
With the auto-update interval set as a duration here would have to be changed to:
```go
AutoUpdateInterval: AutoUpdateInterval.Milliseconds(),
```